incubator/hnc/pkg/validators/hierarchy_test.go

```diff
@@ -143,6 +152,17 @@ func createNS(f *forest.Forest, nnm string, ue []string) *forest.Namespace {
 	return ns
 }
 
+// createRandString generate string for namespace based on given length value
```

Comment:
Can you please take out the randomness and just hardcode one 63-char and one 64-char string? I know it _shouldn't_ make a difference but we shouldn't be testing anything random (we only use randomness on the namespace names to get around the fact that we can't delete them properly in the test context).

Reply:
for this part, i only considered for the purpose that it is easy to check the length of the string exactly 63. rather than counting it manually.

Reply:
Better to just have 64 `a`s IMO then go through this more complicated logic.

Reply:
noted.